docs/components/ActionLabel/ActionLabel.stories.mdx

```diff
@@ -1,7 +1,6 @@
 import { Meta } from "@storybook/addon-docs";
-import { ActionLabel } from "@jobber/components-native";
 
-<Meta title="Components/Actions/ActionLabel" component={ActionLabel} />
```

Comment:
This is not required to render the component in Storybook (as far as I can tell, correct me if I'm wrong), and it causes issues when importing this content in the docs site. Because we're not actually rendering react-native in the docs site (but stubbing it out with react-native-web in the preview window only), it would be a huge pain to get it working at the vite level as well just to not actually render anything in this MDX file.